src/BlazorUI/Bit.BlazorUI/Components/Buttons/ToggleButton/BitToggleButton.scss

```
@@ -21,13 +21,13 @@
     --bit-tgb-icn-margin: #{spacing(0.5)};
 
     &:focus-visible {
+        @include focus-ring(var(--bit-tgb-clr-focus));
         outline-offset: spacing(0.25);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)